modules/apps/27-interchain-accounts/client/cli/cli.go

```diff
@@ -24,3 +24,20 @@ func GetQueryCmd() *cobra.Command {
 
 	return icaQueryCmd
 }
+
+// NewTxCmd returns the tx commands for the interchain-accounts submodule
+func NewTxCmd() *cobra.Command {
+	icaTxCmd := &cobra.Command{
+		Use:                        "interchain-accounts tx",
```

Comment:
```suggestion
		Use:                        "interchain-accounts",
```

it should anyway come as - `simd tx interchain-accounts`

Reply:
ahh nice catch